internal/harness/golang/matchfunction_service.go

```diff
@@ -146,11 +146,9 @@ func (s *matchFunctionService) getMatchManifest(ctx context.Context, req *pb.Run
 
 // TODO: replace this method once the client side wrapper is done.
 func getMMLogicClient(cfg config.View) (pb.MmLogicClient, error) {
+	// Retrieve host name and port. It is ok to not specify host name as there are scenarios
```

Comment:
Should it maybe be set to localhost in that scenario?

Reply:
I think it is ok to not force a null check here - if someone mistakenly does not specify, it fails with connection not being established.